privacy-and-messaging.md

```diff
@@ -0,0 +1,59 @@
+| Content | Description |
+|---|---|
+| Title               | Secure Chat Showdown |
+| Author              | TallAmericano |
+| Featured text       | A Comparative Analysis of Messaging Apps' Privacy and Encryption Features |
+| CT link             | FIXME |
+| Publish date        | 2024-05-20 ?? |
+| Ai checks (https://www.zerogpt.com) | Score must be lower than 25% or article is rejected for publication |
+
+| Social Media | Links |
+|---|---|
+| Bison Relay post    | Title |
+| Reddit link         | FIXME |
+| Tweeter link        | FIXME |
+| TwitterSpace link   | FIXME |
+| Matrix post         | Promote your main tweet in "DCR Social Media" channel |
+
+## Thesis statement
+
+What the article is about, and what is it aiming to prove?
+
+The article aims to provide a comparison of various popular messaging applications in terms of their privacy policies, encryption protocols, and overall security features. 
+
+The central objective is to evaluate how these apps protect user data from unauthorized access, assess their commitment to user privacy, and highlight the strengths and weaknesses of each app's security measures.
+
+
+
+## Pre-discussion, research and questions
+
+* What questions are you aiming to find the answers to?
+
+Which apps provide true encryption and better privacy
+  
+* Show links to research, which statements or quotes are you hoping to use?
+
+
+
+* Who might be a good person to interview for this piece?
+
+Someone from Bison Relay
+
+---
+--- ARTICLE GOES BELOW THIS POINT ---
+
+---
+
+# Secure Chat Showdown
+
+## Outline the types of encryption used by the different popular messaging applications 
+
+## Review privacy policies to determine what data is collected by each application, how it is used, and whether it is shared with third parties.
+
```

Comment:
Are you just looking at messaging app or are you including social media apps like Twitter? If so, You could also look at the algorithms used by these platforms to censor content and users.

In terms of messaging app encryption, Do these platforms provide end to end and server encryption? As I understand it, a lot of the so called end to end encrypted messaging apps don't provide privacy from the server that relays the messages. 

Bison Relay privacy technique:

"Bison Relay privacy is based on a unique combination of a Double Ratchet and post-quantum-secure Public Key Infrastructure. The Double Ratchet creates forward and “reverse” secrecy, meaning a temporary compromise of an endpoint does not compromise all future or all prior message encryption keys. Additionally, the Double Ratchet provides deniable messaging, where either endpoint could have written any message in question. Post-quantum-secure Public Key Infrastructure reinforces the Double Ratchet privacy by protecting it against attacks by quantum computers, and it also makes man-in-the-middle attacks more difficult."

Reply:
Thanks for the comments. Yes I think just messaging apps like element, WhatsApp, signal, bison relay. I will look at how really encrypted the popular ones are. I don’t think I’d be able to expand to twitter etc as would just be information overload. But maybe next article in June could focus on messaging in social media like twitter, Facebook etc

Reply:
Yeah, good point. Nostr might also be a good app, I believe this is mostly a messaging app. You could also do a direct comparison to email systems E.g. why do we need messaging apps when we have email? Email is a pretty broken system at this point, so it shouldn’t be too difficult to build a substantial argument for why we need better privacy preserving messaging apps.